CMakeLists.txt

```diff
@@ -163,7 +167,7 @@ SET (EXECUTABLE_OUTPUT_PATH ${qmcpack_BINARY_DIR}/bin CACHE PATH "Single output
 # QMC_MPI =  enable MPI
 # QMC_OMP = enable OMP
 ######################################################################
-SET(WALKER_MAX_PROPERTIES 32 CACHE STRING "Maximum number of properties tracked by walkers")
```

Comment:
I assume requiring recompiling the code for using more estimators is just a temporary situation.
Cmake doesn't like int https://github.com/QMCPACK/qmcpack/pull/2231

Reply:
Yes this should be temporary.  It should be properly addressed in refactoring the PooledMemory use in Walker,  Estimator and Hamiltonian architecture.

Reply:
Some background info. There are PooledMemory and PooledData used in different places. In the old days, PooledData was the only one used. Since any change on them affects many classes, we'd better discuss what is the new design before making big PRs.